test/FixedGovLst.t.sol

```diff
@@ -2968,3 +3013,19 @@ contract Rescue is FixedGovLstTest {
     assertLe(fixedLst.balanceOf(_holder), _expectedBalance);
   }
 }
+
+contract DelegateeForHolderFixed is FixedGovLstTest {
+  function testFuzz_DelegateeForHolder(address _holder, address _delegatee, uint256 _amount) public {
```

Comment:
Needs a more expressive name for scopelint

Reply:
Done: 3d7a520307fe8a12df63ece44b2671ceaec41bb5